spec/features/user_edits_an_article_spec.rb

```diff
@@ -29,6 +29,9 @@
   end
 
   it "user update their post" do
+    if article
+      10.times { puts "article exists" }
+    end
     visit "/#{user.username}/#{article.slug}/edit"
     fill_in "article_body_markdown", with: template.gsub("true", "false")
```

Comment:
Hmmm "article_body_markdown" is an element that only exists if the user is logged in and has permission to visit the article _and_ has "v1" editor as their editor. Do we know whether they meet all these criteria?